rwcore/platform/FileIndex.cpp

```diff
@@ -39,12 +39,22 @@ void FileIndex::indexTree(const std::filesystem::path &path) {
 }
 
 const FileIndex::IndexedData *FileIndex::getIndexedDataAt(const std::string &filePath) const {
-    auto normPath = normalizeFilePath(filePath);
-    return &indexedData_.at(normPath);
+    std::string normPath = normalizeFilePath(filePath);
+    try {
+	return &indexedData_.at(normPath);
+    }
+    catch (...) {
+	RW_ERROR("Missing file: " << normPath);
+	return nullptr;
+    }
 }
 
 std::filesystem::path FileIndex::findFilePath(const std::string &filePath) const {
-    return getIndexedDataAt(filePath)->path;
+    auto idxData = getIndexedDataAt(filePath);
+    if (!idxData)
+	return filePath;
+
+    return idxData->path;
```

Comment:
Try to follow existing code style. We also have [.clang-format](https://github.com/rwengine/openrw/blob/f10a5a8f7abc79a0728847e9a10ee104a1216047/.clang-format) config to format the code with `clang-format`.